recipe/meta.yaml

```diff
@@ -1,45 +1,53 @@
 {% set name = "tiledbsoma" %}
-{% set version = "1.1.1" %}
+{% set version = "1.1.2" %}
+{% set sha256 = "7c10b39df50b3bf90eaa2237fcfdca349b2d6efdecaef81103979930edd5b1c8" %}
+
+
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
-    git_url: https://github.com/single-cell-data/TileDB-SOMA.git
-    git_rev: b0b91f6d875f71a542859d944aae1fc82c107862
+  #url: https://github.com/single-cell-data/TileDB-SOMA/archive/{{ version }}.tar.gz
+  url: https://github.com/single-cell-data/TileDB-SOMA/archive/64b262c1336c0498ff001d4180d0eb27d1d574cc.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 2
+  skip: true  # [win or linux32 or py2k]
 
 outputs:
   - name: libtiledbsoma
     version: {{ version }}
-    script: build-libtiledbsoma.sh
+    script: build-libtiledbsoma.sh  # [not win]
     requirements:
       build:
+        - {{ compiler('c') }}
         - {{ compiler('cxx') }}
+        - git
         - cmake
-        - make
-        - pybind11
+        - make  # [not win]
       host:
-        - tiledb 2.14.*
+        - tiledb 2.15.*
         - spdlog
+        - fmt
       run:
-        - tiledb 2.14.*
+        - tiledb 2.15.*
         - spdlog
-
+        - fmt
   - name: tiledbsoma-py
     version: {{ version }}
     script: build-tiledbsoma-py.sh
     build:
-      binary_relocation: 
-        - lib/python*/site-packages/tiledbsoma/libtiledbsoma.*
-        - lib/python*/site-packages/tiledbsoma/pytiledbsoma.*.so
     requirements:
       build:
         - {{ compiler('cxx') }}
+        - python                                 # [build_platform != target_platform]
+        - cross-python_{{ target_platform }}     # [build_platform != target_platform]
       host:
```

Comment:
numpy needs to be added to the host requirements and then rerendered to use the numpy pins in `recipe/conda_build_config.yaml`. If pyarrow is also needed during compile time, it should also be added here